model/commitqueue/dequeue_sender.go

```diff
@@ -0,0 +1,66 @@
+package commitqueue
+
+import (
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/message"
+	"github.com/mongodb/grip/send"
+	"github.com/pkg/errors"
+)
+
+type commitQueueDequeueLogger struct {
+	*send.Base
+}
+
+// NewCommitQueueDequeueLogger constructs a Sender that dequeues an item from
+// a commit queue
+func NewCommitQueueDequeueLogger(name string, l send.LevelInfo) (send.Sender, error) {
+	cq := &commitQueueDequeueLogger{
+		Base: send.NewBase(name),
+	}
+
+	if err := cq.SetLevel(l); err != nil {
+		return nil, err
+	}
+
+	cq.SetName(name)
+
+	if err := cq.SetErrorHandler(send.ErrorHandlerFromSender(grip.GetSender())); err != nil {
+		return nil, errors.Wrap(err, "can't set commit queue dequeue error handler")
+	}
+
+	return cq, nil
+}
+
+// Send post issues via jiraCommentJournal with information in the message.Composer
+func (cq *commitQueueDequeueLogger) Send(m message.Composer) {
```

Comment:
if you want this to be testable, I'd refactor this so that you have a doSend method that returns an error, and just call out to the error handler only once.

Reply:
done. (and adjusted the test)